security/expressions.rst

```diff
@@ -12,8 +12,10 @@ Security: Complex Access Controls with Expressions
 In addition to a role like ``ROLE_ADMIN``, the ``isGranted()`` method also
 accepts an :class:`Symfony\\Component\\ExpressionLanguage\\Expression` object::
 
+    // src/Controller/MyController.php
     use Symfony\Component\ExpressionLanguage\Expression;
-    // ...
+
+    // ... your controller code
```

Comment:
do we really need this?

Reply:
hum it is to mimic a class where we extends the sf controller, which gives access to this `denyAccessUnlessGranted`
because we can have a controller without extends, should I instead clearly define a controller class?

Reply:
That makes more sense I guess